### PR TITLE
Add/remove functionality

### PR DESCRIPTION
**Pull request by PrangonGhose**
### Functionalities in this pull request
- Add book title and author name with form submission.
- The book titles and author names are stored in local storage.
- Remove button removes books from the section.
- Remove button removes specific data from local storage.
- Refreshing the page puts the books from local storage in the page if there is any book in local storage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,194 @@
...
+- The project has been made using personalized ideas but following [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) for colors, typography, and layout.
+- The project is done in plane CSS. No CSS framework has been used for styling.
+- Only **Home** and **About** page have been designed as per the requirement of the project.
+- Each of the pages have been adopted for 2 different screen sizes - **Mobile** *768px wide* and **Desktop** *768px or wider* as per the requirement.
+- Interactions (e.g., *Links* and *Mobile menu*) are put.
+- Dynamic page (*Featured Clubs*) is created using JavaScript.
+- Readme file has been adopted as per the requirements.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (ridwanediallo) [inline README.md]: - [ ] Hello team good job so far 👏. However, your readme file does not reflect your project 😄. Please would you like to document it based on the project? Actually, your readme file should only include only the relevant pieces of information that apply to your project. 👌

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @PrangonGhose , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrangonGhose** [comment]
### Corrections after first review
- Update README.md according to the project specifications.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @PrangonGhose , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**